xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml

```diff
@@ -319,6 +319,8 @@ Get the actual notifications.
       .addClass('notification-event-clean')
       .html('$services.icon.renderHTML("trash")&amp;nbsp;$escapetool.xml($escapetool.javascript($services.localization.render('notifications.menu.clear')))')
       .click(function (event) {
+        // Avoid navigating to href="#"
+        event.preventDefault();
```

Comment:
wouldn't that be better to avoid that behaviour by using a button instead of a link?